contrib/systemd/etcd2-backup-coreos/etcd2-join

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+if [ $# -lt 3 ];then
+    echo "USAGE: $0  <master_advertise_peer_urls> <target_name> <target_peer_url>"
+    exit 1
+fi
+
+function convertDropin {
+    sed 's/^Added.*$/[Service]/g'| sed 's/="/=/g' | sed 's/^ETCD_/Environment="ETCD_/g'
+}
+masterAdvUrl=$1
+targetName=$2
+targetUrl=$3
+
+cmd="etcdctl --peers ${masterAdvUrl} member add ${targetName} ${targetUrl}"
+
+set +e
```

Comment:
good point, was at some point going to some post-processing on the error message from etcdctl in-case of non-zero return status from cmd.